starters/gatsby-wordpress-starter/src/templates/page.jsx

```diff
@@ -12,69 +12,24 @@ import "../css/@wordpress/block-library/build-style/theme.css"
 
 import Layout from "../components/layout"
 import Seo from "../components/seo"
+import Page from "../components/page"
```

Comment:
(github won't let me comment on the lines b/c the weren't changed)
Lines 2, 3, and 4 have some imports that can be removed, looks like they're in the component now.